scripts/mppc_to_multiprobe.py

```diff
@@ -0,0 +1,242 @@
+
+    # calculates the distance to the center, returns the distance as one value
+    # - TODO depends on experiments and what is needed
+    # mppc_offset = numpy.linalg.norm(scan_image_center, mppc_cell_center)
+
+    # TODO x and y seem to be swapped somewhere (read/save file?)
+    return mppc_offset
+
+
+def main(args):
+    """
+    Handles the command line arguments.
+    :param args: (list) arguments passed.
+    :returns: (int) TODO value to return to the OS as program exit code
+    """
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--debug', action="store_true", default=False)
+    # nargs="?" to allow to pass just -f without argument, for the Linux desktop
+    # file to work easily.
+    # parser.add_argument('-f', '--file', dest="file_name", nargs="?", default=None,
+    #                     help="Image file to run mppc alignment for")
+    parser.add_argument('--file_offset', dest="file_offset", nargs="?", default="test_mppc_to_multiprobe_offset.h5",
+                        help="Image file to run the mppc to multiprobe offset correction on.")
+    parser.add_argument('--file_rotation', dest="file_rotation", nargs="?", default="test_mppc_to_multiprobe_rotation.h5",
+                        help="Image file to run the mppc to multiprobe rotation correction on.")
+
+    # load image from acquisition server
+
+    options = parser.parse_args(args[1:])
+    if options.debug:
+        logging.getLogger().setLevel(logging.DEBUG)
+    else:
+        logging.getLogger().setLevel(logging.WARNING)
+
+    if not (options.file_offset and options.file_rotation):
+        logging.error("Please specify an offset image and a rotation image for running the mppc to "
+                      "multiprobe alignment procedure.")
+        return
+
+    # read image file
+    try:
+        image_off = hdf5.read_data(options.file_offset)
+        image_offset = image_off[0][0, 0, 0, :, :]
+        image_rot = hdf5.read_data(options.file_rotation)
+        image_rotation = image_rot[0][0, 0, 0, :, :]
+    except Exception as exp:
+        logging.error("%s", exp, exc_info=True)
+        return
+
+    try:
+        # TODO @Job/Andries are these steps independent of each other or not? If not how depending on each other?
+        #  Would need to run the script first on one correction and then do a new scan and run script for second corr..
+        #  If offset not corrected, would the spot position be additionally affected for rotation correction?
+
+        # 1. offset correction
+        # TODO pass only the D4 mppc cell image, all others need to be set black. How do I know where the D4 cell is?
+        # Later only the D4 cell should be active so nothing to worry about
+        mppc_offset = get_mppc_offset(image_offset)
+        print("Distance from image center is x={:.2f} and y={:.2f}. Apply corresponding voltage to the galvo mirror "
```

Comment:
Since we're not converting to a voltage yet add the unit here (pixels I think?).
`...is x={:.2f} pixels and y={:.2f} pixels. ...`